semantic_router/encoders/openai.py

```diff
@@ -20,17 +20,19 @@ def __init__(
         self,
         name: Optional[str] = None,
         openai_api_key: Optional[str] = None,
+        openai_org_id: Optional[str] = None,
         score_threshold: float = 0.82,
         dimensions: Union[int, NotGiven] = NotGiven(),
     ):
         if name is None:
             name = os.getenv("OPENAI_MODEL_NAME", "text-embedding-ada-002")
         super().__init__(name=name, score_threshold=score_threshold)
         api_key = openai_api_key or os.getenv("OPENAI_API_KEY")
-        if api_key is None:
+        org_id = openai_org_id or os.getenv("OPENAI_ORGANIZATION")
```

Comment:
For consistency, I'd rename `"OPENAI_ORGANIZATION"` to `"OPENAI_ORG_ID"`, as the latter is what's used by OpenAI in their code.

Reply:
Great catch. I'm not sure where `OPENAI_ORGANIZATION` came from now, but it is all over my codebase so I will need to promptly update it.